cypress/e2e/pensjon/kalkulator/afp-vs-ufoeretrygd.cy.ts

```diff
@@ -0,0 +1,753 @@
+        cy.get('[data-testid="uttaksgrad"]').select('40 %')
+        cy.get('[data-testid="inntekt-vsa-gradert-uttak-radio-ja"]').check()
+        cy.get('[data-testid="inntekt-vsa-gradert-uttak"]').type('100000')
+        cy.get('[data-testid="age-picker-uttaksalder-helt-uttak-aar"]').select(
+          '62'
+        )
+        cy.get(
+          '[data-testid="age-picker-uttaksalder-helt-uttak-maaneder"]'
+        ).select('1')
+        cy.get('[data-testid="inntekt-vsa-helt-uttak-radio-ja"]').check()
+        cy.get('[data-testid="inntekt-vsa-helt-uttak"]').type('100000')
+        cy.get(
+          '[data-testid="age-picker-inntekt-vsa-helt-uttak-slutt-alder-aar"]'
+        ).select('75')
+        cy.get(
+          '[data-testid="age-picker-inntekt-vsa-helt-uttak-slutt-alder-maaneder"]'
+        ).select('0')
+        cy.contains('Beregn pensjon').click()
+      })
+
+      it('forventer jeg en ingress med informasjon om at "hvis du velger AFP, får du ikke uføretrygd etter at du 62 år. Uføretrygd vises ikke i beregningen"', () => {
+        cy.contains(
+          'Hvis du velger AFP, får du ikke uføretrygd etter at du blir 62 år. Uføretrygd vises ikke i beregningen.'
+        ).should('exist')
+      })
+
+      it('forventer jeg at graf og tabell viser alderspensjon og AFP.', () => {
+        cy.contains('Pensjonsgivende inntekt').should('exist')
+        cy.contains('AFP (avtalefestet pensjon)').should('exist')
+        cy.contains('Pensjonsavtaler (arbeidsgivere m.m.)').should('exist')
+        cy.contains('Alderspensjon (Nav)').should('exist')
+        cy.contains('Tusen kroner').should('exist')
+        cy.contains('61').should('exist')
+        cy.contains('87+').should('exist')
+        cy.contains('Pensjonsavtaler').should('exist')
+        cy.get('[data-testid="showmore-button"]').click()
+        cy.contains('Andre avtaler').should('exist')
+        cy.contains('Privat tjenestepensjon').should('exist')
+        cy.contains('Individuelle ordninger').should('exist')
+        cy.contains('Vis mindre').should('exist')
+      })
+
+      it('forventer jeg ett veilederpanel som anbefaler å ta kontakt for hjelp.', () => {
+        cy.get('[data-testid="vurderer_du_a_velge_afp"]').should('exist')
+      })
+
+      it('forventer jeg informasjon i grunnlag om hvilken AFP som er beregnet.', () => {
+        cy.contains('Øvrig grunnlag for beregningen').should('exist')
+        cy.contains('AFP: Privat').should('exist')
+      })
+
+      it('forventer jeg å kunne endre avanserte valg.', () => {
+        cy.contains('Endre avanserte valg').click()
+        cy.get('[data-testid="uttaksgrad"]').select('50 %')
+        cy.contains('Oppdater pensjon').click()
+        cy.contains('Estimert pensjon').should('exist')
+      })
+    })
+
+    describe('Når jeg velger å beregne Alderspensjon og uføretrygd, uten AFP.', () => {
+      beforeEach(() => {
+        cy.fillOutStegvisning({
+          afp: 'ja_privat',
+          samtykke: true,
```

Comment:
Trenger vel ikke samtykke på afp privat (flere instanser)
```suggestion
          afp: 'ja_privat',
```